### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Diegogagan2587**
# Implemented changes
- Implement styling for the full website

### CODE REVIEW

File: src/components/Navigation.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 function Navigation() {
   return (
     <nav className="navigation">
```
Comment (topeogunleye) [inline src/components/Navigation.js:3]: [OPTIONAL]
- [ ] Your navigation looks great, however, you're missing the user icon at the top right corner of the design. Please kindly add it

image

**Review by topeogunleye — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @Diegogagan2587,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟

wiserhood-cheers

## To highlight :champagne: 
✅ No !important overusing: Avoid excessive use of the !important declaration, as it can lead to specificity and maintainability issues.

✅ No inefficient selectors: Use efficient CSS selectors to target elements without adding unnecessary complexity or performance overhead.

✅ No inline styling: Inline styling should be avoided, except for React inline styling where a variable is created and passed to the React style prop.

## [Optional] suggestions:
- For future pull requests, please notice that your branch name `styling` does not follow the convention of being named after a feature or action. For example, you could name your branch `added-styling`. This makes it easier to track the changes and the purpose of each branch.

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @topeogunleye in your question so I can receive the notification

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navigation.js (hunk 1 above)
Comment (Diegogagan2587) [reply src/components/Navigation.js:3]: Thanks, Will doit, thanks thanks thanks! :D

Comment (topeogunleye) [reply src/components/Navigation.js:3]: You're welcome 🙏

**Pull request merged**